alerting/src/test/kotlin/org/opensearch/alerting/bwc/AlertingBackwardsCompatibilityIT.kt

```diff
@@ -69,7 +71,9 @@ class AlertingBackwardsCompatibilityIT : AlertingRestTestCase() {
                     //  the test execution by a lot (might have to wait for Job Scheduler plugin integration first)
                     // Waiting a minute to ensure the Monitor ran again at least once before checking if the job is running
                     // on time
-                    Thread.sleep(60000)
+                    OpenSearchTestCase.waitUntil({
+                        return@waitUntil false
```

Comment:
Instead of false, we would want to `verifyMonitorStats`. That way we get the benefit of the waitUtil function